movies/src/components/movieEditPopup/movieEditPopup.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import CloseButton from "../closeButton/CloseButton";
+import Modal from 'react-modal';
+import Button from "../common/button/Button";
+import Input from "../common/input/Input";
+import Dropdown from '../common/dropdown/Dropdown';
+import Datepicker from '../common/datepicker/Datepicker';
+import Constants from '../constants';
+import MovieActionPopup from "../movieActionPopup/MovieActionPopup";
+
+export default function MovieEditPopup (props) {
```

Comment:
When modal is opened page content should not be scrollable

Reply:
Fixed

Reply:
![image](https://user-images.githubusercontent.com/16895020/93579528-3dce3400-f9a7-11ea-8da7-534c5d67602b.png)
Still not scrollable

Reply:
Scroll has been added